docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
             context: ./journal
         volumes:
             - ./journal:/srv/app
-            - /public/bundles
+            - /srv/app/public/bundles
```

Comment:
not sure what's inside here, but pointing out it will be persisted between updates to the code (being a volume)

Reply:
Not much in comparison to `vendor`, but would definitely like to find a better solution.